apps/graphql-playground/src/index.css

```diff
@@ -0,0 +1,33 @@
+@import "https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700|Source+Code+Pro:400,700";
```

Comment:
Do we want to host these fonts ourselves? Afaik, google fonts is not GDPR compliant